web/client/codechecker_client/helpers/authentication.py

```diff
@@ -41,6 +41,14 @@ def getAccessControl(self):
     @ThriftClientCall
     def performLogin(self, auth_method, auth_string):
         pass
+    
+    @ThriftClientCall
+    def createLinkGithub(self):
```

Comment:
Same comments as above, should be solved by simplifying the API,

Reply:
fixed